api/rest/restapi_test.go

```diff
@@ -1039,6 +1040,38 @@ func TestAPIRecoverAllEndpoint(t *testing.T) {
 	testBothEndpoints(t, tf)
 }
 
+func TestAPILogging(t *testing.T) {
+	ctx := context.Background()
+	cfg := &Config{}
+	cfg.Default()
+
+	cfg.HTTPLogFile = "/tmp/http.log"
```

Comment:
set to a relative path. There's nothing wrong, but cluster tests do not write anything outside the local repository folder.